spec/abstract-jobs.html

```diff
@@ -202,6 +212,19 @@ <h1>Execution</h1>
           in a FinalizationGroup do not necessarily hold that FinalizationGroup live.
         </p>
       </emu-note>
+      <emu-note>
+        <p>
+          Implementations are not obligated to empty WeakRefs for maximal sets
```

Comment:
This note is good, but I think it might be clearer if you gave an example, explaining how this works out with a single object, and how it allows cycles to be collected. But this could be done in a follow-on patch.